packages/gatsby-transformer-code-examples/gatsby-node.js

```diff
@@ -0,0 +1,93 @@
+const createSchemaCustomization = ({ actions, schema }) => {
+  const { createTypes } = actions;
+
+  const typeDefs = [
+    schema.buildObjectType({
+      name: 'CodeExample',
+      fields: {
+        name: 'String!',
+        extension: 'String!',
+        absolutePath: 'String!',
+        content: 'String!',
+      },
+      interfaces: ['Node'],
+      extensions: {
+        infer: false,
+      },
+    }),
+  ];
+
+  createTypes(typeDefs);
+};
+
+async function onCreateNode({
+  node,
+  actions,
+  loadNodeContent,
+  createNodeId,
+  createContentDigest,
+}) {
+  if (!isValidNode(node)) return;
```

Comment:
Should we maybe log a warning or something if the node is not valid?

Reply:
There are many node types used by Gatsby internally or by other plugins that are not valid for this plugin - I don't think we should log those.

Reply:
True, make sense then